lua/autorun/server/sv_votestuff.lua

```diff
@@ -0,0 +1,53 @@
+if !SERVER then return end
+
+util.AddNetworkString( "tv::SendTasks" )
+util.AddNetworkString( "tv::SendAnswer" )
+
+Seconds = { // Each X seconds will appear a vote for all online players. They will have X/2 seconds to anwser;
+	10,
+} 
+GivePoints = 100; // Amount of points to give the winner;
+Curr_Task = Curr_Task || {}
+
+// Creating a random timer on init;
+local uniqueID = 'FetchQuestions'
+local time = Seconds[math.random(1, #Seconds)]
+timer.Create(uniqueID, time, 0, function()
+	if !timer.Exists(uniqueID) then timer.Remove(uniqueID) return; end;
```

Comment:
line is unused (and makes no sense, you're removing something you know doesn't exist)